.gitignore

```diff
@@ -1,10 +1,13 @@
 # Files created by the pipeline, which we want to keep out of git
 # (or at least out of _this_ git repo).
-data/
-results/
-auspice/
-build/
-
+*data/
+*results/
+*auspice/
+*build/
+*results/
```

Comment:
You don't need `results` in here twice. Personally, I like to keep stuff like this ASCIIbetically sorted because it makes duplicatons like this jump out. 

```suggestion
```

Reply:
Ohh, I like the ASCIIbetically idea. Thanks for catching